.gitignore/githubToken.js

```diff
@@ -0,0 +1 @@
+var githubToken = '26634634f2f95cf221b3faddd0a9774e6eb33392';
```

Comment:
this looks right in atom (a file named .gitignore with githubToken.js in the body) but shouldn't be showing up here in Github -- you may want to reset this token, since it's very tough to remove once pushed.